kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -209,11 +209,6 @@ public LogSegment getLogSegmentForVersion(Engine engine, Optional<Long> versionT
   public LogSegment getLogSegmentForVersion(
       Engine engine, Optional<Long> versionToLoadOpt, List<ParsedLogData> parsedLogDatas) {
     final long versionToLoad = versionToLoadOpt.orElse(Long.MAX_VALUE);
-
-    // Defaulting to listing the files for now. This has low cost. We can make this a configurable
-    // option in the future if we need to.
-    final boolean USE_COMPACTED_FILES = true;
```

Comment:
IMO we don't need this. It's not configurable today, and if we ever want to make it configuration then it's trivial to undo this change.